psi4/include/psi4/psifiles.h

```diff
@@ -280,6 +280,11 @@
 
 #define PSIF_SAD                 300  /*- A SAD file (File for SAD related quantities -*/
 
+#define PSIF_DLPNO_QAB_PAO       310  /*- A PSIO file to store all (AUX | PAO PAO) integrals in a DLPNO computation -*/
+#define PSIF_DLPNO_QIA_PNO       311  /*- A PSIO file to store all (AUX | LMO PNO) integrals in a DLPNO computation -*/
+#define PSIF_DLPNO_QAB_PNO       312  /*- A PSIO file to store all (AUX | PNO PNO) integrals in a DLPNO computation -*/
+#define PSIF_DLPNO_TRIPLES       313  /*- A PSIO file to store all X_{ijk}^{abc} like terms in a DLPNO-CCSD(T) computation -*/
```

Comment:
also for py-side https://github.com/psi4/psi4/blob/master/psi4/driver/psifiles.py#L212

Reply:
Done!